dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/datafile/ingest/IngestReport.java

```diff
@@ -52,8 +57,12 @@ public void setId(Long id) {
     @JoinColumn(nullable = false)
     private DataFile dataFile;
 
-    @Lob
-    private String report;
+    @Enumerated(EnumType.STRING)
+    private IngestError errorKey;
+
+    @ElementCollection
+    @OrderColumn
+    private List<String> reportArguments = new ArrayList<>();
```

Comment:
If we renamed the `report` field to `errorKey`, than this one could maybe be `errorArguments`?